Marlin/src/lcd/marlinui.cpp

```diff
@@ -1509,9 +1509,6 @@ void MarlinUI::update() {
       wait_for_heatup = wait_for_user = false;
       card.flag.abort_sd_printing = true;
     #endif
-    #ifdef ACTION_ON_CANCEL
-      host_action_cancel();
-    #endif
```

Comment:
This is needed to tell the host to cancel the job when there is no SD card. It won't be invoked anywhere else.